.github/workflows/ubuntubuilds.yml

```diff
@@ -0,0 +1,76 @@
+name: Ubuntu builds
+
+on:
+  push:
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  vanilla:
+    name: Vanilla Ubuntu 20.04
+    runs-on: ubuntu-latest
+    container:
+      image: 337955887028.dkr.ecr.us-east-2.amazonaws.com/kpsr-docker-registry/github/kpsr-core:Vanilla_1.0.0
+      credentials:
+        username: AWS
+        password: ${{ secrets.ECR_PASSWORD }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Build
+        run:
+          mkdir build && cd build && cmake -DKPSR_WITH_DOXYGEN=true -DKPSR_WITH_DDS=false -DKPSR_WITH_ZMQ=false \
```

Comment:
- Change all builds to Release
- Add one that does `KPSR_TEST_PERFORMANCE=true` for this, you will need google benchmarks.

Reply:
CMake flags for Coverage are only added when build is in Debug mode. So it might be necessary to have "Release" and Debug build both.

Reply:
In that case we can change all current jobs to build in Release mode, and add one job for coverage and further linters in Debug.